waku/v2/node/peer_manager/peer_manager.nim

```diff
@@ -333,6 +337,7 @@ proc new*(T: type PeerManager,
                        storage: storage,
                        initialBackoffInSec: initialBackoffInSec,
                        backoffFactor: backoffFactor,
+                       outPeersTarget: max(maxConnections div 10, 10),
```

Comment:
yup, the eclipse attack was also my concern. took this ratio from both nimbus (1:9) and prysm (1:8) but we can revisit later on.
* [nimbus](https://github.com/status-im/nimbus-eth2/blob/40253a76ddcfd2aa29ef662bb8ee61306093fd1c/beacon_chain/networking/eth2_network.nim#L1559)
* [prysm](https://github.com/prysmaticlabs/prysm/blob/537236e1c9cbecf863a391a7b5f097313b8043c0/beacon-chain/p2p/peers/status.go#LL68C2-L68C14)

unsure how they justify this number given the possibility of eclipse attacks. Perhaps from a practical point of view its more complex since the "eclipser" is compeeting with the whole network to get these slots, so it would be very difficult for the attacker to end up having all slots?

Reply:
Probably unrelated to the eclipse attack, but would it make sense to make to ratio configurable?